gocd/bci_repo_publish.py

```diff
@@ -152,8 +174,16 @@ def run(self, version, token=None):
             self.logger.info('No positive result from openQA (yet)')
             return
 
+        if rebuild_token:
+            for pkg in self.fetch_package_names_in_project(_DEVEL_PRJ_APIURL, devel_prj):
```

Comment:
jftr OBS team will fix this that we can use unscoped tokens to trigger project wide rebuilds

Reply:
implementation is ready at  https://github.com/openSUSE/open-build-service/pull/15119

Reply:
> So shall we wait for them to implement this before moving this forward?

no, I think we can do this afterwards. however I'd really like to get one repo published first before adding more risk of regressions..